### PR TITLE
feat: refactoring to the new schema

### PR DESCRIPTION
**Pull request by AlexsJones**
```
❯ grpcurl -plaintext  localhost:8080 list 
grpc.reflection.v1.ServerReflection
grpc.reflection.v1alpha.ServerReflection
schema.v1.ServerAnalyzerService
schema.v1.ServerConfigService

```

This PR is pinned against `buf.build` remote schema update to [this](https://buf.build/k8sgpt-ai/k8sgpt/docs/volitile-01:schema.v1) version

## ✅ Checks

- [ x] My pull request adheres to the code style of this project
- [x ] My code requires changes to the documentation
- [x ] I have updated the documentation as required
- [x ] All the tests have passed

### CODE REVIEW

File: pkg/custom/client.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ func NewClient(c Connection) (*Client, error) {
 
 func (cli *Client) Run() (common.Result, error) {
 	var result common.Result
-	req := &schemav1.AnalyzerRunRequest{}
+	req := &schemav1.RunRequest{}
```
Comment (matthisholleville) [inline pkg/custom/client.go:34]: Shouldn't we rename this to `RunAnalyzeRequest` ?

**Review by matthisholleville — CHANGES_REQUESTED** [review]
Great work ! Left 1 comment/suggestion and the linter report 2 failure checks

File: pkg/custom/client.go (hunk 1 above)
Comment (AlexsJones) [reply pkg/custom/client.go:34]: I could update this in an update?

**Comment by AlexsJones** [comment]
> Great work ! Left 1 comment/suggestion and the linter report 2 failure checks

Thanks, update

**Review by JuHyung-Son — APPROVED** [review]
LGTM!

**Pull request merged**